app/wrappers/wakes/facebook_metrics_wrapper.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+require 'json'
+require 'httparty'
+
+class Wakes::FacebookMetricsWrapper
+  attr_reader :url
+
+  def initialize(url)
+    @url = url
+  end
+
+  def total_count
+    metrics['total_count']
+  end
+
+  def metrics
+    parsed_response
+  end
+
+  private
+
+  # rubocop:disable Metrics/MethodLength
+  def parsed_response
+    attempts = 0
+    begin
+      make_request.parsed_response[0]
+    rescue => e
+      puts "Failed #{attempts} attempt(s) - #{e}"
+      attempts += 1
+      if attempts <= times
```

Comment:
@benhutton: I noticed that I haven't tested this part of the code. `times` used here isn't defined. I'll add in tests for error handling.